core/src/miner/miner.rs

```diff
@@ -592,6 +600,15 @@ impl Miner {
     }
 }
 
+impl TimeoutHandler for Miner {
+    // Dummy timeout handler
+    fn on_timeout(&self, timer: TimerToken) {
+        match timer {
```

Comment:
Use `unimplemented!()`.
And I cannot find a reason to use matching `timer`.

Reply:
It is for future usage.

Reply:
I think it's redundant. I'll delete them.